test/e2e/flask/multichain-api/testHelpers.ts

```diff
@@ -93,6 +104,44 @@ export async function getSessionScopes(
   return JSON.parse(await getSessionRawResult.getText());
 }
 
+/**
+ * Use dapp UI to send custom addresses to request scope for.
+ *
+ * @param driver - E2E test driver {@link Driver}, wrapping the Selenium WebDriver.
+ * @param accounts - The addresses to get session scope for.
+ */
+export async function addRequestAccountsToCreateSession(
```

Comment:
`addAccountsToCreateSessionForm`? The naming makes me initially think that this method changes the account checkboxes in the wallet_createSession approval itself

Reply:
resolved in [de15bfb](https://github.com/MetaMask/metamask-extension/pull/29356/commits/de15bfb58aa908ad3b7f54092a123b53e963a270)